Source/Editor/SceneGraph/Actors/SplineNode.cs

```diff
@@ -407,6 +468,60 @@ internal static void OnSplineEdited(Spline spline)
             }
         }
 
+        private static SplinePointNode SelectedSplineNode()
+        {
+            var selection = Editor.Instance.SceneEditing.Selection;
+            if (selection.Count != 1)
+                return null;
+            if (selection[0] is not SplineNode.SplinePointNode)
+                return null;
+
+            return (SplinePointNode)selection[0];
+        }
+
+        private static List<Spline> GetSplinesOnView()
+        {
+            var splines = Level.GetActors<Spline>();
+            var splinesOnView = new List<Spline>();
+
+            var viewTransform = Editor.Instance.Windows.EditWin.Viewport.ViewTransform;
+            var viewFov = Editor.Instance.Windows.EditWin.Viewport.FieldOfView;
+            var viewNear = Editor.Instance.Windows.EditWin.Viewport.NearPlane;
+            var viewFar = Editor.Instance.Windows.EditWin.Viewport.FarPlane;
+            var viewAspect = Editor.Instance.Windows.EditWin.Width / Editor.Instance.Windows.EditWin.Height;
```

Comment:
Create a BoundingFrustum needs a aspect ratio view but i don't know how get this value, because this, the frustum don't work correctly and sometimes don't return correctly splines on view

Reply:
Does it work if you do the Viewport Width/ Viewport Height?